src/Components/ArtworkFilter/ArtworkFilterSavedSearchAlertContextProvider.tsx

```diff
@@ -0,0 +1,33 @@
+import { usePrepareFiltersForPills } from "Components/ArtworkFilter/Utils/usePrepareFiltersForPills"
+import {
+  SavedSearchAlertContextProvider,
+  SavedSearchAlertContextProviderProps,
+} from "Components/SavedSearchAlert/SavedSearchAlertContext"
+import { getSearchCriteriaFromFilters } from "Components/SavedSearchAlert/Utils/savedSearchCriteria"
+import { FC } from "react"
+
+interface ArtworkFilterSavedSearchAlertContextProviderProps
+  extends Omit<SavedSearchAlertContextProviderProps, "criteria"> {}
+
+/**
+ * A simple component that just wraps the `SavedSearchAlertContextProvider` in order
+ *  to omit the `criteria` prop, which is inferred from the filter context
```

Comment:
The criteria this provides isn't used. The provider pushes a copy of it into it's own state, but keeps the name the same. I believe it's so that your alert criteria and search criteria can diverge. Which is fine. But it is confusing.